.github/workflows/ci.yml

```diff
@@ -43,14 +43,6 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-cargo-index-
 
-      - name: Cache cargo build
-        uses: actions/cache@v2.1.7
-        with:
-          path: target
-          key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-cargo-build-target-
-
```

Comment:
@simonsso we need to restore this or if it was the reason for running out of space, we'd need to remove all the cache steps plus the place where we are checking for a cache hit.

Reply:
I did not see any significant speedup from this, but I will reconsider.

Reply:
Speed up is only for when we have a cache hit